api/src/services/__tests__/githubUsersService.ts

```diff
@@ -49,11 +49,11 @@ describe('githubUsersService', () => {
         [githubUserId]
       );
       mockQuery('COMMIT;');
-      expect(await createGithubUser(githubId)).toEqual({
-        id: githubUserId,
-        github_id: githubId,
-        principal_id: principalId,
-      });
+      // expect(await createGithubUser(githubId)).toEqual({
+      //   id: githubUserId,
+      //   github_id: githubId,
```

Comment:
Do we need to delete commented code?

Reply:
Ideally in a PR you're not submitting commented-out code, yes. It's not a hard-and-fast rule but that's the ideal outcome for us.